client/components/FEAuth.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from "../styles/FEAuth.module.css";
+import Button from "../components/Button";
+
+const FEAuth = () => {
```

Comment:
May as well just call it auth, it's implicit that it's a frontend component

Reply:
Done 👍